clarifai/rag.py

```diff
@@ -0,0 +1,122 @@
+from datetime import datetime
+
+import yaml
+from google.protobuf.struct_pb2 import Struct
+
+from clarifai.client.app import App
+from clarifai.client.model import Model
+from clarifai.client.user import User
+from clarifai.client.workflow import Workflow
+
+
+class RAG:
+  """
+    RAG is a class for Retrieval Augmented Generation.
+
+    Example:
+        >>> from clarifai.rag import RAG
+        >>> rag_agent = RAG()
+    """
```

Comment:
Doc strings indentation is inconsistent with the rest of SDK

Reply:
Pre-commit hooks all passed.